### ISSUE TITLE
False positive `Missing npm packages:` when installing from tarball

### ISSUE THREAD

**Issue by jacobq** [quaertym/ember-cli-dependency-checker#89]
I've run into a problem that I suspect is related to `ember-cli-dependency-checker`:
https://github.com/felixrieseberg/ember-electron/pull/355#issuecomment-404622536

Basically we are installing a package via the [`npm install <tarball file>` syntax](https://docs.npmjs.com/cli/install) (to work around a caching problem and a symlink problem), but the dependency checker seems to get confused about this and think the wrong package is installed: 

```
Missing npm packages: 
Package: ember-electron
  * Specified: file:../../tmp-2580rXGPRrApjWO8/ember-electron-cachebust.tgz
  * Installed: file:C:/Users/appveyor/AppData/Local/Temp/1/tmp-2580rXGPRrApjWO8/ember-electron-cachebust.tgz
Run `npm install` to install missing dependencies.
```

**Comment by jacobq** [comment]
@quaertym Would you mind releasing as 2.2.1? (That way I don't need to modify my `package.json` to point to a branch/commit on GH.)
